### PR TITLE
Add optimade

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
I added a pull request upstream to include the license in the pip package https://github.com/Materials-Consortia/optimade-python-tools/pull/594

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/optimade/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  entry_points:
+    - optimade-validator=optimade.validator:validate
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - email_validator ~=1.1
```
Comment (ocefpaf) [inline recipes/optimade/meta.yaml:25]: I'm not sure conda supports the `~=` syntax. You'll have to expand that. In this case, if my PEP440 is not failing me, that should be >=1.1,<1.2.

Comment (jan-janssen) [reply recipes/optimade/meta.yaml]: I had the feeling it was working during the tests, but I am also fine with just using `>=`.

Comment (jan-janssen) [reply recipes/optimade/meta.yaml]: But then `pip check` complains.

**Comment by jan-janssen** [comment]
`pygraphviz` is not available for windows but I would still prefer to release the packages as `no arch`.

**Comment by jan-janssen** [comment]
@ocefpaf Can you take another look?

**Pull request merged**